var/spack/repos/builtin/packages/libdrm/package.py

```diff
@@ -6,13 +6,18 @@
 from spack import *
 
 
-class Libdrm(AutotoolsPackage):
+class Libdrm(Package):
     """A userspace library for accessing the DRM, direct rendering manager,
     on Linux, BSD and other systems supporting the ioctl interface."""
 
     homepage = "https://dri.freedesktop.org/libdrm/"
-    url      = "https://dri.freedesktop.org/libdrm/libdrm-2.4.59.tar.gz"
+    list_url = "https://dri.freedesktop.org/libdrm/"
```

Comment:
```suggestion
    url      = ( "https://dri.freedesktop.org/libdrm/libdrm-2.4.100.tar.gz, " +
                 "https://dri.freedesktop.org/libdrm/libdrm-2.4.101.tar.xz" )
```
`list_url` didn't work with url_for_version, and is not needed when using this set of urls. Now `spack versions libdrm` works again.

 @alalazo is this a stable approach?

Reply:
```suggestion
    url      = "https://dri.freedesktop.org/libdrm/libdrm-2.4.101.tar.xz"
```
Nevermind, this is sufficient to catch updates; no need to overthink it.